### PR TITLE
DrugSearch.java

### PR DESCRIPTION
**Pull request by Rammurthy123**
example java code

### CODE REVIEW

**Comment by Rammurthy123** [comment]
example

**Review by banoth522 — COMMENTED** [review]
remove the commented code

**Comment by Rammurthy123** [comment]
ok

**Comment by banoth522** [comment]
ok

File: reviewBranch [hunk 1]
```diff
@@ -0,0 +1,76 @@
+package com.uhg.optumrx.controller.validation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.lang.StringUtils;
```
Comment (banoth522) [inline reviewBranch:6]: remove commented code

**Review by banoth522 — COMMENTED** [review]
Leave a comment

**Pull request closed without merging**